### DISCUSSION TITLE
news/20240206/

### DISCUSSION THREAD

**Discussion by giscus** [CommandPost/FCPCafe#338] [Website Discussions]
# news/20240206/

George Edmondson from Seed Creative(and the guy from all the MotionVFX tutorials!) has discounted his 

https://fcp.cafe/news/20240206/

**Comment by simplevideoedit** [comment]
I bought this years ago and it never really worked quite right. This prompted me to email the developer because my old purchase link doesn't work anymore but I haven't received a response for about a week now. If it hasn't been updated though, I doubt I would see any change.
